object/user_util.go

```diff
@@ -140,21 +141,26 @@ func setUserProperty(user *User, field string, value string) {
 	}
 }
 
-func SetUserOAuthProperties(organization *Organization, user *User, providerType string, userInfo *idp.UserInfo) (bool, error) {
+func SetUserExternalIdpProperties(user *User, provider *Provider, userInfo *idp.UserInfo) (bool, error) {
+	providerCategory := strings.ToLower(provider.Category)
 	if userInfo.Id != "" {
-		propertyName := fmt.Sprintf("oauth_%s_id", providerType)
+		propertyName := fmt.Sprintf("%s_%s_id", providerCategory, provider.Type)
 		setUserProperty(user, propertyName, userInfo.Id)
+
+		_, err := uuid.Parse(userInfo.Id)
```

Comment:
Are you sure that userInfo.Id at any provider has uuid format?

Reply:
if ```userInfo.Id``` won't be in uuid format, it just won't be updated

Reply:
why?

Reply:
we have ability to map id, so we should map it, but user can set attribute that won't be in uuid format, so we won't map it. or u'd like to throw err, when userInfo.Id is not in uuid format?

Reply:
I think sync logic shouldn't depends on id format